radix-common/src/lib.rs

```diff
@@ -87,6 +87,7 @@ pub mod prelude {
 
 pub(crate) mod internal_prelude {
     pub use super::prelude::*;
+    pub use crate::*;
```

Comment:
I believe this creates an infinite import possibilities loop which slowed rust-analyzer to a crawl before?

Can we instead be more precise about what to import?

Reply:
Good point! I've resolved this.